.env.example

```diff
@@ -2,8 +2,23 @@ APP_NAME=app
 CLUSTER=conn
 GIT_REPO_NAME="ce-connector-ditto"
 GIT_BASE_BRANCH=main
+HOST_APP_PORT=3000
 
+# app
+APP_VERSION=1
+GIT_COMMIT_SHA=sha
 APP_PORT=3000
-APP_ENV=production
+APP_BIND_ADDRESS=0.0.0.0
+APP_ENV=development
+LOG_LEVEL=info
+API_ADDRESS_DITTO=https://api.dittowords.com
```

Comment:
🟠 We have `docker/.env.default` and this one (`.env.example`). Ideally we should use the one in the `docker/` and update the README.  We can use [Iterable](https://github.com/lokalise/ce-connector-iterable/blob/main/README.md) as an example.

Reply:
fixed